contracts/Removal.sol

```diff
@@ -43,12 +45,56 @@ contract Removal is
   }
 
   /**
-   * @dev mints all of the removal vintages for an issuance
+   * @dev returns the removal vintage data for a given removal token ID
+   */
+  function vintage(uint256 removalId) public view returns (Vintage memory) {
+    return _vintages[removalId];
+  }
+
+  /**
+   * @dev returns the token ids for a set of removals given each one's vintage and the data that was provided
+   * in its mint transaction
+   */
+  function tokenIdsForRemovals(
+    bytes32[] memory parcelIdentifiers,
+    uint256[] memory removalVintages
+  ) public view returns (uint256[] memory) {
+    require(
+      parcelIdentifiers.length == removalVintages.length,
+      "parcelIdentifers and removalVintages must be same length"
```

Comment:
```suggestion
      "Removal: ..."
```

Although it seems weird to have a revert in a public view function.

Reply:
I think you're right - I did a little research and it sounds like while this would work as expected in Remix or using a local ganache chain, it won't actually revert on a public network.  I pulled this out and instead added to the documentation to make it clear that these arrays should be the same length because they correspond to the same group of removals.